src/sentry/snuba/metrics_layer/api.py

```diff
@@ -0,0 +1,328 @@
+import re
```

Comment:
We talked about this in Slack, but I don't think this belongs in the `snuba` folder. Part of this is a naming issue we should resolve.

Metrics API: The external metrics API
Existing Metrics Layer: The metrics layer that uses DerivedMetrics etc.
New Metrics Layer/API?: The new layer/API that Enoch and I are working on. I was calling it the metrics API to distinguish it from the metrics layer above but now that is confusing.

I think it's important to keep the hierarchy clear: The API calls the new layer. The new layer is also not specific to the API (it will be used by all features that use metrics). All the code for this API should live outside of the snuba folder.

Reply:
Yep! I will definitely find a better place for it